boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash -x
+
+usage() { echo "Usage: $0 -o operator_name -i operator_image -c saas-repository-channel [-t]" 1>&2; exit 1; }
+
+# TODO : Add support of long-options 
+while getopts "c:dg:h:i:n:o:" option; do
+    case "${option}" in
+        c)
+            operator_channel=${OPTARG}
+            ;;
+        d)
+            diff_generate=true
+            ;;
+        g)
+            generate_script=${OPTARG}
+            ;;
```

Comment:
Script is not expected to be used manually (it can but not the standard usecase which is to use make target) and within code, I would prefer avoiding the default - still, I will add check on the values